src/themes/teams/components/Segment/segmentStyles.ts

```diff
@@ -1,14 +1,23 @@
-import { ICSSInJSStyle } from '../../../types'
+import { SegmentProps } from 'semantic-ui-react'
+import { ICSSInJSStyle, ComponentSlotStylesInput } from '../../../types'
 import { SegmentVariables } from './segmentVariables'
 
-export default {
-  root: ({ variables }: { variables: SegmentVariables }): ICSSInJSStyle => {
+const segmentStyles: ComponentSlotStylesInput<SegmentProps, SegmentVariables> = {
+  root: ({ props: { inverted }, variables: v }): ICSSInJSStyle => {
     return {
-      padding: variables.padding,
-      background: variables.background,
-      border: '1px solid rgba(34,36,38,.15)',
-      borderRadius: variables.borderRadius,
-      boxShadow: '0 1px 2px 0 rgba(34,36,38,.15)',
+      padding: v.padding,
+      background: v.background,
+      borderRadius: v.borderRadius,
+      boxShadow: '0 1px 1px 1px rgba(34,36,38,.15)',
+      ...(inverted
+        ? {
+            background: v.color,
```

Comment:
Is it okay that the inverted segment doesn't have border color? Should be consider having two variables for colors (color and borderColor) and switch there when the inverted prop is applied?

Reply:
discussed with @mnajdova  this is according to spec